### DISCUSSION TITLE
Is it possible to read models in the GetStaffPicksAsync list without logging in?

### DISCUSSION THREAD

**Discussion by kyousuke916** [pixiv/vroid-sdk-developers#75] [Q&A]
Hi, I would like to ask, I am currently using VRoidSDK-v0.1.4.
I would like to know if there is a way to use DefaultApi.GetStaffPicksAsync() to get a list of recommended models and read them without logging in?
I am testing and found that I can use the DefaultApi.GetStaffPicksAsync() API to get a list of recommendations without logging in.
However, there is an error in reading the model using the staff pick list of data.

```
Usage
List<StaffPicksCharacterModel> characterModels = await defaultApi.GetStaffPicksAsync(count);

var go = await ModelLoader.LoadVrmAsync(characterModels[0].character_model,
(progress) =>
{
    Log($"progress:{progress}");
});
```

Error message
Pixiv.VroidSdk.ModelLoadFailException: {"error":"invalid_request","error_description":"Missing required parameter: refresh_token."}

**Comment by nyamadan** [comment]
HI.

> I would like to know if there is a way to use DefaultApi.GetStaffPicksAsync() to get a list of recommended models and read them without logging in?

There is no way to use GetStaffPicksAsync without logging in.
All of DefaultApi requires login.
